### PR TITLE
Pegasus M13 experiments runner

### PR DESCRIPTION
**Pull request by spigo900**

### CODE REVIEW

**Comment by spigo900** [comment]
This should be complete, but I think it makes sense to hold off on merging it until #949 is merged since that makes changes to the M13 runner as well.

File: adam/experiment/__init__.py
Comment (LichMaster98) [inline adam/experiment/__init__.py:183]: This error message looks incomplete.

**Review by LichMaster98 — CHANGES_REQUESTED** [review]

File: adam/experiment/__init__.py
Comment (spigo900) [reply adam/experiment/__init__.py:183]: And it is. :)

Thanks for catching this.

Comment (spigo900) [reply adam/experiment/__init__.py]: This should be fixed now.

**Review by LichMaster98 — APPROVED** [review]

**Pull request merged**
